cmd/kube-scheduler/app/options/options.go

```diff
@@ -280,9 +291,15 @@ func (o *Options) Config() (*schedulerappconfig.Config, error) {
 	return c, nil
 }
 
-func kubeconfigFileExists(name string) bool {
-	_, err := os.Stat(name)
-	return err == nil
+func kubeconfigFileExists(kubeConfigFilenames string) ([]string, bool) {
```

Comment:
Nit. probably rename it as parseKubeconfigList ?

Reply:
Yeah, it's a nit as it was old code :) Will do